packages/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     { "path": "vc-api-issuer" },
     { "path": "vc-api-verifier" },
     { "path": "vc-handler-ld-local" },
-    { "path": "did-auth-siop-op-authenticator" }
+    { "path": "did-auth-siop-op-authenticator" },
+    { "path": "vc-api-verifier-plugin" },
```

Comment:
dont think this is the correct path as it should be "vc-api-verifier", which is already in there

Reply:
It was result of a bad merge. updated

Reply:
i see no change

Reply:
https://github.com/Sphereon-Opensource/ssi-sdk/blob/7ae753ca78785a3b13f8a84cc4e713a70daf4db2/packages/tsconfig.json